roles/edpm_nova_compute/defaults/main.yml

```diff
@@ -186,7 +187,7 @@ edpm_nova_compute_libvirt_enabled_perf_events: ''
 edpm_nova_compute_libvirt_inject_key: false
 edpm_nova_compute_libvirt_inject_partition: '-2'
 edpm_nova_compute_libvirt_inject_password: false
-edpm_nova_compute_libvirt_live_migration_inbound_addr: '{{ hostvars[inventory_hostname]["fqdn_" ~ service_net_map["nova_libvirt_network"]] }}'
+edpm_nova_compute_libvirt_live_migration_inbound_addr: '{{ hostvars[inventory_hostname]["fqdn_" ~ edpm_service_net_map["nova_libvirt_network"]] }}'
```

Comment:
if we were keeping this role longterm this is somehting we woudl want to review.
in general we expect this to be an ip not an fqdn but both will work.
i assume the fqdn was used because of TLSe so this is not nessisarly wrong just not what we would normally expect to be popluatled

https://docs.openstack.org/nova/latest/configuration/config.html#libvirt.live_migration_inbound_addr

if you notice the upstream docs detail that this should be an ip addres

IP address used as the live migration address for this host.
"""
This option indicates the IP address which should be used as the target for live migration traffic when migrating to this hypervisor. This metadata is then used by the source of the live migration traffic to construct a migration URI.

If this option is set to None, the hostname of the migration target compute node will be used."""

im not entirly sure if '{{ hostvars[inventory_hostname]["fqdn_" ~ edpm_service_net_map["nova_libvirt_network"]] }}'
is returning an ip or an fqdn but if its the former and it works its technially relying on undocumented behavior

This is prexisting behavior and not related to this PR however just wanted to note that while reviewing.